POC/AzureVision.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Nice overview. Thanks!</p><p> </p><p>Would be interested in seeing the quality of the text that is too small and or blurry. Indeed, could be as simple as cropping and zooming on text labels, and perhaps running a simple sharpening filter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-herbarium/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Looked at a JPEG. The image seems much more blurry than the original. Let's talk about how it was scaled.